src/pages/ballot/index.tsx

```diff
@@ -34,20 +34,22 @@ const ClearBallot = () => {
     <>
       <div
         className="cursor-pointer text-gray-400 underline hover:text-black"
-        onClick={() => setOpen(true)}
+        onClick={() => {
```

Comment:
useCallback and don't use click handlers with div (should be button in this case)